--- a/nv-acpi.c
+++ b/nv-acpi.c
@@ -300,7 +300,11 @@ static int nv_acpi_remove(struct acpi_de
     if (pNvAcpiObject->notify_handler_installed)
     {
         // no status returned for this function
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,6,0)
+        acpi_os_wait_events_complete();
+#else
         acpi_os_wait_events_complete(NULL);
+#endif
 
         // remove event notifier
         status = acpi_remove_notify_handler(device->handle, ACPI_DEVICE_NOTIFY, nv_acpi_event);
