diff -ur a/conftest.sh b/conftest.sh
--- a/conftest.sh
+++ b/conftest.sh
@@ -1781,10 +1781,6 @@
             CODE="
             #include <asm/atomic.h>
             void conftest_atomic64(void) {
-                atomic64_t data;
-                atomic64_read(&data);
-                atomic64_set(&data, 0);
-                atomic64_inc(&data);
             }"
 
             compile_check_conftest "$CODE" "NV_ATOMIC64_PRESENT" "" "types"
